internal/local/app_test.go

```diff
@@ -725,6 +726,12 @@ exports = function({ query }) {
 	},
 }}}
 
+var fullMeta = MdbAppMeta{
```

Comment:
do we have tests for `local.LoadAppMeta`?

Reply:
We should also add tests for local.FindApp to check 
- app with ./mdb/app_meta.json, loads ./mdb/app_meta.json (verify contents)
- app without ./mdb/app_meta.json doesn't load anything
for both cases, we should still load the original config